project/conf/base/rest_framework.py

```diff
@@ -10,4 +10,7 @@
         "rest_framework.renderers.BrowsableAPIRenderer",
         "rest_framework_csv.renderers.PaginatedCSVRenderer",
     ],
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.TokenAuthentication',
```

Comment:
i would remove it from the default auth classes.

Reply:
@mattiagiupponi , I removed it.